Description: XDG_RUNTIME_DIR is unreliable on CentOS 7.x:
	it is created on login (even SSH logins),
	but not when using su.

Index: xpra/platform/xposix/paths.py
===================================================================
--- a/xpra/platform/xposix/paths.py	(revision 13756)
+++ b/xpra/platform/xposix/paths.py	(working copy)
@@ -10,7 +10,7 @@
 
 from xpra.util import envbool
 
-USE_RUNTIME_DIR = envbool("XPRA_USE_RUNTIME_DIR", True)
+USE_RUNTIME_DIR = envbool("XPRA_USE_RUNTIME_DIR", False)
 
 USE_RUNTIME_LOG_DIR = envbool("XPRA_USE_RUNTIME_LOG_DIR", USE_RUNTIME_DIR)
 USE_RUNTIME_BIN_DIR = envbool("XPRA_USE_RUNTIME_BIN_DIR", USE_RUNTIME_DIR)
